clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerNetworkThreadTest.java

```diff
@@ -231,10 +254,7 @@ public void testAssignmentChangeEvent() {
 
         consumerNetworkThread.runOnce();
         verify(applicationEventProcessor).process(any(AssignmentChangeEvent.class));
-        verify(networkClient, times(1)).poll(anyLong(), anyLong());
-        verify(commitRequestManager, times(1)).updateAutoCommitTimer(currentTimeMs);
-        // Assignment change should generate an async commit (not retried).
-        verify(commitRequestManager, times(1)).maybeAutoCommitAsync();
+        verify(networkClientDelegate, times(1)).poll(anyLong(), anyLong());
```

Comment:
can we be consistent at using times(1) here? above you removed times(1) but it seems to be inconsistently applied elsewhere.